.github/workflows/integration-tests.yml

```diff
@@ -157,6 +157,7 @@ jobs:
     permissions:
       contents: write
     needs:
+    - configuration
```

Comment:
hmm, interesting. I expect this is an important fix, although, as usual, I'm disappointed in the documentation...

https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context

> The needs context contains outputs from all jobs that are defined as a dependency of the current job. For more information on defining job dependencies, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idneeds)."

Technically the `github-release` job is defined as dependent on the `linux-arm64` job, and that job `needs: configuration`. But it seems likely that this isn't what the developers thought when they implemented it.

Reply:
Indeed, this does appear to be a critical bit, as demonstrated by: https://github.com/jsoref/cascading-needs/actions/runs/3079668494

Reported as https://github.com/github/docs/issues/20701#issue-1377326735